e2e/tests/permissions.js

```diff
@@ -12,11 +12,11 @@ describe('Permissions', () => {
     const CreatePage = createPageFactory(
         'http://localhost:8083/#/users/create'
     )(driver);
-    const EditPage = editPageFactory('http://localhost:8083/#/users/1/edit')(driver);
+    const EditPage = editPageFactory('http://localhost:8083/#/users/1')(driver);
```

Comment:
nope, since your change on show/edit default.

Reply:
it should not be commited